.github/pull_request_template.md

```diff
@@ -13,6 +13,6 @@
 ## Change checklist
 
 - [ ] Self-review.
-- [ ] Documentation updates if relevant.
+- [ ] Documentation updates following the [style guide][https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#appendix-a-full-conventions-text], if relevant.
```

Comment:
```suggestion
- [ ] Documentation updates following the [style guide](https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#appendix-a-full-conventions-text), if relevant.
```
markdown link fix

Reply:
Also maybe this is a nicer-to-read link to the same text:
https://rust-lang.github.io/rfcs/1574-more-api-documentation-conventions.html#appendix-a-full-conventions-text

Reply:
updated the link, thanks